spec/lib/locasms/numbers_spec.rb

```diff
@@ -5,74 +5,102 @@
 
   describe '.initialize' do
     subject do
-      LocaSMS::Numbers.any_instance
-        .should_receive(:evaluate)
-        .once
-        .with([:numbers])
-        .and_return({ good: [1,3], bad: [2,4] })
+      expect_any_instance_of(LocaSMS::Numbers).
+        to receive(:evaluate).
+        once.
+        with([:numbers]).
+        and_return(good: [1,3], bad: [2,4])
       LocaSMS::Numbers.new :numbers
     end
 
-    it{ subject.good.should == [1,3] }
-    it{ subject.bad.should == [2,4]  }
+    it { expect(subject.good).to eq([1,3]) }
+    it { expect(subject.bad).to eq([2,4]) }
   end
 
   describe '#normalize' do
-    it{ subject.normalize('+55 (11) 8888-9999').should == %w(551188889999) }
-    it{ subject.normalize('55', ['11', '22']).should == %w(55 11 22) }
-    it{ subject.normalize(['55', 'ZZ', '22']).should == %w(55 ZZ 22) }
-    it{ subject.normalize('55,44,33', ['ZZ', '22,11']).should == %w(55 44 33 ZZ 22 11) }
-    it{ subject.normalize(55, [11, 22]).should == %w(55 11 22) }
-    it{ subject.normalize('Z').should == ['Z'] }
-    it{ subject.normalize(nil).should == [] }
+    it do 
+        expect(subject.normalize('+55 (11) 8888-9999')).to(
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.